tasks/apt.yml

```diff
@@ -1,6 +1,10 @@
 - name: Update APT package cache
```

Comment:
@hispanico 
Someone recently informed me it's not a good practice to include `update_cache` in role, rather it's better to add it in playbook directly like:

```
  pre_tasks:
    - name: update apt cache
      apt:
        update_cache: yes
        cache_valid_time: 86400
```
Reason:
Updating the package cache is very time consuming (you can even get blocked if you do it too frequently). For that reason it doesn't belong in individual roles, it should be done only once at the start of the playbook.

Can you please comment on the same? Should we change accordingly?

Thanks

Reply:
I'm not familiar with this, but I have seen that updating the package cache is time consuming. The change sounds beneficial to me.

Reply:
I prefer if we leave this how it is now, I'd fix this role and then start focusing the devops technical effort the dockerization project